### ISSUE TITLE
Combined servo endpoints, reversing, subtrim etc

### ISSUE THREAD

**Issue by Eurylokhos** [FrSky-OS/discussion#47]
Currently all the various servo setup parameters are on separate screens (endpoints, subtrim, and reversing). This makes setting up a new model more difficult and cluttered, most newer radios have these functions integrated in a single screen. It makes for a more streamlined and easier setup.

**Comment by janekx** [comment]
Yes it would be better to have in one page

**Comment by yellowbird911** [comment]
Some radios have separate menus for subtrim, direction and end point but they are very unpleasant to program.   My vote also would be to have all servo set up functions on one page.

**Comment by AloftHobbies** [comment]
Yes, one page! It is pretty much the most important page for me when setting up a molded glider. I assign channels then limit the endpoints to avoid damage to plane or servos.
